rpm/toggle-monitor/toggle-monitor.spec

```diff
@@ -0,0 +1,24 @@
+Name:     toggle-monitor
+Version:  1
+Release:  1
+Summary:  Script to turn monitor on and off
+License:  Public Domain
+
+Requires: xrandr
+
+SOURCE0:  monitor-toggle
+SOURCE1:  10-keybindings-toggle-monitor
+
+%description
+RPM package with toggle-monitor script
+
+%install
+install -d %{buildroot}/usr/local/bin
+install -m 755 %{SOURCE0} %{buildroot}/usr/local/bin/monitor-toggle
+install -d %{buildroot}/etc/dconf/db/local.d
+install %{SOURCE1} %{buildroot}/etc/dconf/db/local.d/10-keybindings-toggle-monitor
+
+%files
+/usr/local/bin/monitor-toggle
```

Comment:
As we are installing it as a package, it should go into /usr/bin. And maybe it could be good to use macro here instead of /usr/bin.

Reply:
Changed path to /usr/bin